API proposal- Radio signal strength API (for rainfall analysis  use cases)

Kamran-Keykhosravi opened the pull request:
#### What type of PR is this?

* enhancement/feature
API proposal


#### What this PR does / why we need it:

Adds the API proposal (API-proposal-Attenuation_Rainfall.md) and slides for presentation (API-proposal-Attenuation_Rainfall.pptx)


#### Which issue(s) this PR fixes:

<!-- Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`. -->

Fixes #267 

#### Special notes for reviewers:



#### Changelog input

```
documentation/API proposals/API-proposal-Attenuation_Rainfall.md
        documentation/SupportingDocuments/API-proposal-Attenuation_Rainfall.pptx

```

#### Additional documentation 

This section can be blank.



```
docs

```

Kamran-Keykhosravi committed:
added api and support document for issue 261- rainfall

Kamran-Keykhosravi commented:
> * EasyCLA check passed. You are authorized to contribute.

Hello Alberto,

Now the CLA issue is seemed to be solved at my end. Can you please check and let me know if I should do anything else.

Bests,
Kamran.

Kamran-Keykhosravi commented:
Hello @albertoramosmonagas . 
We are working on modifying this API proposal based on the comments. I was just wondering if I should create a new pull request for the revised API description or do something else.

albertoramosmonagas commented on documentation/API proposals/API-proposal-Attenuation_Rainfall.md:
```diff
@@ -0,0 +1,47 @@
+### API family owner  
+Ericsson
+
+### API summary  
```
Hi @Kamran-Keykhosravi, as a suggestion for improving the wording of this paragraph, I would propose the following:
- First sentence: what the API provides in plain language (“near real-time rainfall data using the operator's network”).
- Second: how it does this at a high level (it uses radio links between antennas that are affected by rain).
- Third: what it is used for (rain models, forecasting, drones, etc.).

Kamran-Keykhosravi committed:
Update API-proposal-Attenuation_Rainfall.md

Addressing comments in November meeting

Kamran-Keykhosravi committed:
Update and rename API-proposal-Attenuation_Rainfall.md to API-proposal-Strength_Rainfall.md

Renamed the file and also the API family name by changing attenuation to strength.

albertoramosmonagas commented:
@camaraproject/api-backlog_codeowners - We need a review. Since the backlog session on December 11, there have been no objections (https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/389414929/2025-12-11+API+Backlog+Minutes). The idea is to send it to the next TSC on December 18.

jgarciahospital reviewed (approved):
LGTM

albertoramosmonagas reviewed (approved):
LGTM
